code/modules/reagents/chemistry_reagents/other.dm

```diff
@@ -973,6 +973,18 @@
 	objective_value = OBJECTIVE_EXTREME_VALUE
 	properties = list(PROPERTY_BIOCIDIC = 2)
 
+/datum/reagent/plasma/hemocatalene
+	name = "Hemocatalene Plasma"
+	id = PLASMA_HEMOCATALENE
+	description = "A red-ish plasma..."
+	color = "#cf5b51"
+	overdose = REAGENTS_OVERDOSE
+	overdose_critical = REAGENTS_OVERDOSE_CRITICAL
+	chemclass = CHEM_CLASS_SPECIAL
+	objective_value = OBJECTIVE_EXTREME_VALUE
+	properties = list(PROPERTY_HEMOGENIC = 20, PROPERTY_REPAIRING = 2, PROPERTY_TOXIC = 1)
```

Comment:
"the latter providing the target with increased blood regeneration, **brute damage healing**, and toxin damage."
Repairing provides healing only to cybernetic limbs, not brute in general. What you're looking for is Neogenetic (NGN)